pkg/console/operator/operator.go

```diff
@@ -217,17 +217,15 @@ func (c *consoleOperator) handleSync(configs configSet) error {
 		return fmt.Errorf("console is in an unknown state: %v", updatedStatus.Spec.ManagementState)
 	}
 
-	// set default conditions ok first, sync can toggle if conditions are invalid
-	c.ConditionsDefault(updatedStatus)
```

Comment:
Needed to remove this one to prevent the hotlooping. Other approach would be to remove the `v1helpers.SetOperatorCondition()` that is setting the `Degraded` status type inside the `ConditionsDefault()`

Reply:
Nice catch. thinking on this one with you.

Reply:
`Degraded` is the one with most potential to jitter, since its affected by other controllers we setup in `starter.go`.

Reply:
Holding on this one as we consider the approach. I think we should just go ahead and use the same helper func method as in the PR you linked  https://github.com/openshift/cluster-authentication-operator/pull/142#issue-293463578.

So we probably want something like:

```Golang
handleDegraded(operatorConfig, "CustomLogoInvalid", ...) // Creates DegradedCustomLogoInvalid
```
The old helpers can slowly be replaced by this method then.  The `NewClusterOperatorStatusController()` will then gather all of the `Degraded` variants and determine (automatically) if we are Degraded.  And the same for Available, etc.

Reply:
We can probably do these one by one in small PRs.

Reply:
`handledDegraded()` could be a good first one to pull over & update.